index.html

```diff
@@ -2094,12 +2199,33 @@ <h2>
           A boolean that indicates whether the <a>user agent</a> SHOULD collect
           and return the billing address associated with a <a>payment
           method</a> (e.g., the billing address associated with a credit card).
-          Typically, the user agent will return the billing address as part of
-          the {{PaymentMethodChangeEvent}}'s <a>methodDetails</a>. A merchant
-          can use this information to, for example, calculate tax in certain
-          jurisdictions and update the displayed total. See below for privacy
-          considerations regarding <a href="#user-info">exposing user
-          information</a>.
+          The <a>user agent</a> returns the billing address as part of the
+          {{PaymentMethodChangeEvent}}'s <a>methodDetails</a> attribute and the
+          {{PaymentResponse}}'s <a>details</a> attribute. A payee can use this
+          information to, for example, calculate tax in certain jurisdictions
+          and update the displayed total. See below for privacy considerations
+          regarding <a href="#user-info">exposing user information</a>.
```

Comment:
```suggestion
          regarding [[[#user-info]]].
```

You might need to reword this one after it expands out. However, avoid `<a href=>` and let ReSpec deal with linking :)